src/NuGet/Microsoft.Net.Compilers.Toolset/Microsoft.Net.Compilers.Toolset.Package.csproj

```diff
@@ -9,9 +9,14 @@
     <DevelopmentDependency>true</DevelopmentDependency>
     <PackageDescription>
       .NET Compilers Toolset Package.
-      Referencing this package will cause the project to be built using the specific version of the C# and Visual Basic compilers contained in the package, as opposed to any system installed version.
+      Referencing this package will cause the project to be built using the C# and Visual Basic compilers contained in the package, as opposed to the version installed with MSBuild.
+
+      This package is primarily intended as a method for rapidly shipping hot fixes to customers. Using it as a long term solution for providing newer compilers on older MSBuild installations is explicitly not supported. That can and will break on a regular basis.
+
+      The supported mechanism for providing new compilers in a build enviroment is updating to the newer .NET SDK or Visual Studio Build Tools SKU.
+
+      This package requires MSBuild 16.3 and .NET Desktop 4.7.2+ or .NET Core 2.1+
```

Comment:
The and/or grouping here is ambiguous since "either" got dropped.

Reply:
I didn't feel like the "either" really helped but can add it back.